jdi-light/src/main/java/com/epam/jdi/light/actions/ActionHelper.java

```diff
@@ -286,15 +290,15 @@ static void afterAction(ActionObject jInfo, Object result) {
         TIMEOUTS.element.reset();
     }
     private static void waitAfterAction(ActionObject jInfo) {
-        IBaseElement element = jInfo.element();
+        JDIBase element = jInfo.element();
         if (element == null) return;
-        Pair<String, Integer> waitAfter = element.base().waitAfter();
+        Pair<String, Integer> waitAfter = element.waitAfter();
         if (isNotBlank(waitAfter.key) && jInfo.methodName().equalsIgnoreCase(waitAfter.key) && waitAfter.value > 0) {
             Timer.sleep(waitAfter.value * 1000);
         }
     }
-    public static JFunc2<ActionObject, Object, Object> AFTER_STEP_ACTION = ActionHelper::afterStepAction;
-    public static JFunc2<ActionObject, Object, Object> AFTER_JDI_ACTION = ActionHelper::afterJdiAction;
+    public static JAction2<ActionObject, Object> AFTER_STEP_ACTION = ActionHelper::afterStepAction;
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6397461)